src/intl/en/learn-quizzes.json

```
@@ -107,15 +107,15 @@
   "b004-c-explanation": "This answer is partially correct, but it's only one of the reasons ETH is valuable.",
   "b004-d-label": "All of the above",
   "b004-d-explanation": "Ethereum transactions cannot be censored, ETH is required to make any transaction on Ethereum, and it is crucial to the stability of the DeFi ecosystem.",
-  "c001-prompt": "Web3 allows users to own digital assets directly through:",
-  "c001-a-label": "DAOs",
-  "c001-a-explanation": "DAOs (Decentralized autonomous organizations) are member-owned communities without centralized leadership.",
+  "c001-prompt": "Web3 allows users to own digital assets through:",
+  "c001-a-label": "Tokens",
+  "c001-a-explanation": "Tokens provide a way to represent assets with the same properties as an Ethereum-based asset. Tokens are not unique. While they represent ownership, there are more ways to own digital assets on Ethereum.",
```

The wording here confused me a little...  "with the same properties as an Ethereum-based asset" and "Tokens are not unique"...

Maybe:

> Tokens provide a way to represent units of value that are interchangeable with one another, owned by an Ethereum account.

```suggestion
  "c001-a-explanation": "Tokens provide a way to represent units of value that are interchangeable with one another, owned by an Ethereum account. While they represent ownership, there are more ways to own digital assets on Ethereum.",
```